test/unit/RelayServer.test.ts

```diff
@@ -347,6 +349,24 @@ describe('RelayServer', () => {
                 `Wrong fees receiver address: ${fakeFeesReceiverAddress}\n`
             );
         });
+
+        it('should throw error if request is expired', () => {
```

Comment:
shouln't we need to add another test if the request is not expired?

Reply:
All other test cases work because their requests are not expired :)